adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala

```diff
@@ -69,6 +70,17 @@ object ADAMMain extends Logging {
       )
     )
 
+  def main(args: Array[String]) {
+    new ADAMMain(defaultCommandGroups).apply(args)
```

Comment:
You can just make defaultCommandGroups the default value in the `ADAMMain` constructor, which will simplify this to `new ADAMMain()(args)`